Fix clang error by fixing the asm file emitted by luajit,
https://crbug.com/1056849.
Github PR: https://github.com/LuaJIT/LuaJIT/pull/562

diff --git a/src/vm_x86.dasc b/src/vm_x86.dasc
index 2ccc671..2db6735 100644
--- a/src/vm_x86.dasc
+++ b/src/vm_x86.dasc
@@ -6182,7 +6182,7 @@ static void emit_asm_debug(BuildCtx *ctx)
     fprintf(ctx->fp, "\t.section .eh_frame,\"aw\",@progbits\n");
 #endif
 #else
-    fprintf(ctx->fp, "\t.section .eh_frame,\"a\",@progbits\n");
+    fprintf(ctx->fp, "\t.section .eh_frame,\"a\",@unwind\n");
 #endif
     fprintf(ctx->fp,
 	".Lframe1:\n"
